doc/edl/README.md

```diff
@@ -3,16 +3,30 @@
 ## Background
 
 A PaddlePaddle training job contains several trainer instances,
-several parameter server instances, and one master instance. We would
-like to automatically scale the number of training instances and the
+several parameter server instances, and one master instance. 
```

Comment:
Background section should only tell us about "Why we need to do this", designs and thoughts need to put to bottom sections.